classes/class_springarm3d.rst

```diff
@@ -23,6 +23,13 @@ Description
 
 .. rst-class:: classref-reftable-group
 
+Tutorials
+---------
+
+- :doc:`Third-person camera with spring arm <../tutorials/3d/spring_arm>`
+
+.. rst-class:: classref-reftable-group
+
```

Comment:
These changes need to be made in the XML files in the main repo. More info [here](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html#the-source-of-the-class-reference)

If you can't figure that out, feel free to just remove the changes from this file, and the link can be added after this PR is approved and merged

Reply:
Will make sure to add the link there after this is merged 🫡